.ci/azure-pipelines/env.yml

```diff
@@ -42,30 +42,23 @@ jobs:
   strategy:
     matrix:
       Ubuntu 18.04:
-        CUDA_VERSION: 10.2
-        UBUNTU_DISTRO: 18.04
-        USE_CUDA: true
+        # Test baseline with 18.04
+        UBUNTU_VERSION: 18.04
         VTK_VERSION: 6
+        ENSENSOSDK_VERSION: 2.3.1557
         TAG: 18.04
       Ubuntu 20.04:
-        CUDA_VERSION: 11.2.1
-        UBUNTU_DISTRO: 20.04
+        # Test the latest LTS version of Ubuntu
```

Comment:
Perhaps, we should rename the names on L44, 50 to Ubuntu LTS (oldest) and Ubuntu LTS (latest)

Reply:
"oldest" may be misleading as this would suggest Ubuntu 4.10 ;-)

Reply:
😆 So not oldest, but a suitable non-number adjective. A bit of a mouthful suggestions:
* Oldest supported
* Oldest non-EOL
* Oldest maintained
* Oldest active

Reply:
Okay, so:
- 18.04: `Test the oldest supported version of Ubuntu`
- 20.04: `Test the latest LTS version of Ubuntu`
- 20.10: Drop from the list or keep in case someone want test sth with this version?
- 21.04: `Test the latest version of Ubuntu`

Reply:
20.10 can be dropped in favor of 21.04

Reply:
ok, comments adjusted